tests/data/quartznet_test.yaml

```diff
@@ -0,0 +1,110 @@
+model: "QuartzNet"
+sample_rate: 16000
+dropout: &drop 0.2
+
+AudioToTextDataLayer:
+    max_duration: 16.7
+    trim_silence: true
+
+    train:
+        shuffle: true
+
+    eval:
+        shuffle: false
+        max_duration: null
+
+AudioPreprocessing:
```

Comment:
You are going to have to change this in your new PR.

Reply:
Yep, can do.